### DISCUSSION TITLE
Are there any links to active plugin repositories?

### DISCUSSION THREAD

**Discussion by jazir555** [SteamDeckHomebrew/decky-loader#54] [Q&A] [answered]
I've seen a few different plugin loaders, but have not yet been able to find a good repository for plugin downloads. Does anyone have some good resources for Steam Deck plugins?

**Comment by Fisch03** [comment]
You can find (and install) all plugins that have been submitted [here](https://plugins.deckbrew.xyz/)!

**Comment by marios8543** [comment, marked as answer]
https://plugins.deckbrew.xyz is the legacy one that will be replaced soon. Useful for finding plugins that are not on the new one
https://beta.deckbrew.xyz is the new and improved store. Will become the main one soon.

Open either in Google Chrome in gaming mode and click on the install button. You should get a prompt from the plugin loader to install the plugin.
